spec/cli_spec.rb

```diff
@@ -73,6 +73,80 @@
 - + - + -
 6 | 7 | 8
 
+GAMEOUTPUT
+
+    end
+  end
+
+  context "a game that ends in a draw" do
+    let(:input) { StringIO.new("0\n1\n4\n2\n5\n3\n6\n8\n7") }
+    let(:output) { StringIO.new }
+
+    it "can play a game" do
```

Comment:
This test isn't quite specific enough I think, it's a matter of preference but if you want to test the code you specifically wrote in `cli` I'd be looking for ways to just test the end of the game, how would you do that?